### ISSUE TITLE
Combo and selection doesn't work

### ISSUE THREAD

**Issue by msansm1** [sadfuzzy/redcase#14]
Hi,

I have upgraded my redmine and redcase from an old version (redmine : from 1.3.2 to 2.3.3 and redcase from 0.148 to 1.8) and during the process I have any error returned.
When in a test case I try to change execution suite or select a test to see the result nothing works. The change of value for any combo box has no effect, and the selection of a test does the same...

I have nothing on the console on my server, but I have "ReferenceError: Form is not defined" on the firebug console when I go to "test case" tab on redmine.

If you have an idea or need more information, just ask :)

Thanks

**Comment by AndreMiras** [comment]
Yes this is broken Since Ruby on Rails  (and Redmine 2.1) moved from Prototype to jQuery.
I guess all the JavaScript needs to be migrated.
